recipes/pylogit/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "pylogit" %}
+{% set version = "0.2.2" %}
+{% set sha256 = "a6809e2adb3063fe1408696080217440ab6ea5eb1f3aab62eba9e215559e2fbc" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - future
+    - numpy
+    - pandas
+    - scipy
+    - statsmodels
+    - tqdm
+
+test:
+  imports:
+    - pylogit
+
+about:
+  home: https://github.com/timothyb0912/pylogit
+  license: BSD-3-Clause
+  license_family: BSD
```

Comment:
Please manually package the license file:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

and include a link to the following PR as a comment in the recipe:

https://github.com/timothyb0912/pylogit/pull/18

Reply:
I have added this.